src/libcip54.ts

```diff
@@ -949,7 +1512,17 @@ export const getFile = async (
     }
     if (!file) {
       try {
+/**
```

Comment:
Comment not needed for inline function, or at least I wouldn't comment it as a function, I would comment it as a one-liner just using //